cmd/emp/certs.go

```diff
@@ -37,10 +34,7 @@ func init() {
 }
 
 func runCertAttach(cmd *Command, args []string) {
-	if len(args) == 0 {
```

Comment:
Looks like this one breaks previous behavior (note `==` rather than `!=` like the rest). I think `NumArgs` should just be 1, instead of 0.